daemons/ipa-kdb/ipa_kdb_mspac.c

```diff
@@ -80,6 +80,21 @@ static char *memberof_pac_attrs[] = {
 #define AUTHZ_DATA_TYPE_PAD "PAD"
 #define AUTHZ_DATA_TYPE_NONE "NONE"
 
+krb5_error_code filter_logon_info(krb5_context context,
+                                  TALLOC_CTX *memctx,
+                                  krb5_data realm,
+                                  struct PAC_LOGON_INFO_CTR *info);
+void get_authz_data_types(krb5_context context, krb5_db_entry *entry,
```

Comment:
I think these should be declared in ipa_kdb_mspac_private.h, since the only reason they're not static is so that they can be used in ipa_kdb_tests.c